eth/ethconfig/config.go

```diff
@@ -215,7 +215,8 @@ type Config struct {
 	// OverrideTerminalTotalDifficultyPassed (TODO: remove after the fork)
 	OverrideTerminalTotalDifficultyPassed *bool `toml:",omitempty"`
 
-	RollupSequencerHTTP string
+	RollupSequencerHTTP     string
+	RollupHistoricalRPCFlag string
```

Comment:
I'd generally expect this to be `RollupHistoricalRPC` instead of `RollupHistoricalRPCFlag` (i.e. drop the `Flag`) b/c it is too similar to the CLI flag variable name.

Reply:
Good point, I made that change.